Makefile

```diff
@@ -58,6 +58,11 @@ checksum:
 clean:
 	@rm -rf $(BINDIR) ./_dist
 
+dev:
+	go generate $(GOFILES)
```

Comment:
This is just a target I made for compiling for quick iteration, I used it as a convenience and thought others would find it useful. It just compiles, and generates, without testing